.github/workflows/packaging.yaml

```diff
@@ -94,3 +97,87 @@ jobs:
         run: |
           ./packaging/msi/winget_manifest.sh > Tetrate.func-e.yaml
           yamllint -sd '{extends: default, rules: {line-length: disable}}' Tetrate.func-e.yaml
+
+      - name: "Upload artifacts"
+        if: runner.os == 'Linux'
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+
+  linux-x86:
+    name: "Test installing on (${{ matrix.image }}/${{ matrix.arch }})"
+    needs: dist
+    runs-on: "ubuntu-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image: "ubuntu:latest"
+            arch: amd64
```

Comment:
let's use arch here to represent the docker arch, not a filename expression, as the mapping is confusing.